src/archivematicaCommon/lib/storageService.py

```diff
@@ -285,11 +285,11 @@ def create_file(uuid, origin_location, origin_path, current_location,
         else:
             url = _storage_service_url() + 'file/'
             response = session.post(url, json=new_file)
-    except requests.exceptions.RequestException as e:
+        # raise an HTTPError exception if getting HTTP 4xx or 5xx error
+        response.raise_for_status()
+    except (requests.exceptions.RequestException, requests.exceptions.HTTPError) as e:
```

Comment:
`requests.RequestException` is the base class in Request library so we're already catching all the subclasses too. See https://github.com/requests/requests/blob/master/requests/exceptions.py.